package.json

```diff
@@ -5,6 +5,9 @@
   "files": [
     "lib"
   ],
+  "bugs": {
+    "url": "https://github.com/americanexpress/react-albus/issues"
+  },
   "main": "lib",
   "scripts": {
```

Comment:
use commitlint as part of the test/posttest

Reply:
@asutedja here is an example https://github.com/americanexpress/eslint-config-amex/blob/master/package.json#L10-L11

Reply:
Updated in 37289a5